topdown/crypto.go

```diff
@@ -41,3 +42,67 @@ func builtinCryptoX509ParseCertificates(a ast.Value) (ast.Value, error) {
 func init() {
 	RegisterFunctionalBuiltin1(ast.CryptoX509ParseCertificates.Name, builtinCryptoX509ParseCertificates)
 }
+
+// createRootCAs creates a new Cert Pool from scratch or adds to a copy of System Certs
+func createRootCAs(tlsCaCertFile string, tlsCaCertEnvVar []byte, tlsUseSystemCerts bool) (*x509.CertPool, error) {
+
+	var newRootCAs *x509.CertPool
+
+	if tlsUseSystemCerts {
+		systemCertPool, err := x509.SystemCertPool()
+		if systemCertPool == nil {
```

Comment:
❓ This looks a bit odd. Did you not do
```go
systemCertPool, err := x509.SystemCertPool()
if err != nil {
	return nil, err
}
```
on purpose? Right now, it looks like this could `return nil, nil`, if `systemCertPool` is `nil` and `err` is, too. But I haven't peeked under the hood, I don't know what could be returned here 🤔